GCTC-NTGC/gc-digital-talent
# Tooling - 4027 - Add storybook dark mode

Resolves #4027 

> **Note**
> This was in the safebox but I already had most of it stashed, just needed to wire things up.

## Summary

This adds a global control to storybook for our `ThemeProvider` for dark mode 🌔 .

## Screenshot

<img width="165" alt="Screen Shot 2022-10-06 at 1 02 38 PM" src="https://user-images.githubusercontent.com/4127998/194374731-618c888b-69d9-456c-bf2a-9b2efd40463e.png">

## Testing

> **Note**
> The best stories to test this are "ThemeSwitcher" and "Home Page" (GC Digital, not IAP).

1. Startup storybook `npm run storybook`
2. Confirm the global control exists in the header
3. Confirm it actually changes the theme mode
